src/routes/transactions/entities/swap-order-info.entity.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  TransactionInfo,
+  TransactionInfoType,
+} from '@/routes/transactions/entities/transaction-info.entity';
+import {
+  ApiExtraModels,
+  ApiProperty,
+  ApiPropertyOptional,
+} from '@nestjs/swagger';
+
+export class TokenInfo {
+  @ApiPropertyOptional()
```

Comment:
```suggestion
  @ApiPropertyOptional({ type: Number, nullable: true })
```

Reply:
Changed in [e3a4a9c](https://github.com/safe-global/safe-client-gateway/pull/1299/commits/e3a4a9c33aac2d31572dd3cb51e6cacec9ed3515) (should be `String` though).

Reply:
Oops, `String`, yes 😄

Thanks!